### DISCUSSION TITLE
marimo notebooks runs in browser via WASM does not need python right?

### DISCUSSION THREAD

**Discussion by umeshkacha** [marimo-team/marimo#3180] [Q&A] [answered]
Hi I am new to WASM and notebooks I want to confirm marimo notebooks runs in browser via WASM does not need python right? Python installation at os/system path leve does it need? Can someone please confirm?

**Comment by mscolnick** [comment, marked as answer]
Yes, that is correct. Using the WebAssembly notebook (like https://marimo.app/, or the [static export](https://docs.marimo.io/guides/exporting.html#export-to-wasm-powered-html)) does not require Python to be installed locally.

**Comment by umeshkacha** [comment]
Thanks, How/where do we deploy an app kind of https:// marimo.app so
customer can use it in their site? Thanks again.

**Reply by akshayka** [reply]
@umeshkacha, you can use our newly released feature that lets you export marimo notebooks as interactive static HTML. We have preliminary docs here: https://docs.marimo.io/guides/exporting.html#export-to-wasm-powered-html
